utils/Trackers.h

```diff
@@ -43,6 +43,8 @@ class AvgRateTracker
 
     void update(double value, uint64_t timestamp);
     double get() const;
+    double get(uint64_t timestamp, uint64_t maxSilence) const;
```

Comment:
I understand what the "maxSilence" is designed for, but consider changing the name to be less 'audio' and more generic. E.g. maxTimeWithOutSamples (or something shorter).